db/memtable.cc

```diff
@@ -106,7 +106,8 @@ MemTable::MemTable(const InternalKeyComparator& cmp,
       insert_with_hint_prefix_extractor_(
           ioptions.memtable_insert_with_hint_prefix_extractor),
       oldest_key_time_(std::numeric_limits<uint64_t>::max()),
-      atomic_flush_seqno_(kMaxSequenceNumber) {
+      atomic_flush_seqno_(kMaxSequenceNumber),
+      fragmented_tombstones_list(nullptr) {
```

Comment:
Do you mind renaming it `fragmented_tombstones_list_` to conform with the instance variable naming convention?